src/bridge/ExposureNotification/ExposureNotificationAdapter.android.ts

```diff
@@ -33,5 +40,18 @@ export default function ExposureNotificationAdapter(exposureNotificationAPI: Exp
       }
       return [];
     },
+    getExposureWindowsAndroid: async (diagnosisKeysURLs: string[]) => {
+      captureMessage('getExposureWindows');
+      await exposureNotificationAPI.provideDiagnosisKeys(diagnosisKeysURLs);
+      const _exposureWindows = await exposureNotificationAPI.getExposureWindows();
+      const exposureWindows = _exposureWindows.map(window => {
```

Comment:
Having two variables, `_exposureWindows` and `exposureWindws` seems unnecessarily confusing. How about just  returning the results of the `.map()` call instead of creating the 2nd variable?

Reply:
Yep good idea.